qemu/tests/qemu_guest_agent.py

```diff
@@ -2279,6 +2279,61 @@ def gagent_check_user_logoff(self, test, params, env):
         error_context.context("Verify if guest agent works.", logging.info)
         self.gagent_verify(self.params, self.vm)
 
+    @error_context.context_aware
+    def gagent_check_blacklist(self, test, params, env):
+        """
+        Verify the blacklist of config file, linux guest only
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment.
+        """
+        session = self._get_session(params, None)
+        self._open_session_list.append(session)
+
+        error_context.context("Try to execute guest-file-open command which"
+                              " is in blacklist by default.", logging.info)
+
+        ranstr = utils_misc.generate_random_string(5)
+        guest_file = "/tmp/qgatest" + ranstr
+        try:
```

Comment:
better to put this large try block into a function.

Reply:
large try block?
or you mean make a function for the two try block operation?